README.md

```diff
@@ -299,31 +255,14 @@ Pre-built binaries of this module are provided for the following versions of NGI
 
-## Testing
-
-To test this module, you'll need the [Test::Nginx Perl module](https://github.com/openresty/test-nginx) and [docker-compose](https://docs.docker.com/compose/install/) installed. 
-
-To run the tests do the following:
-
-* run `./configure`, make sure you select _No_ for Dynamic module 
-* run `make`
-* Edit the resources/test/run.sh script and set the `ADMIN_PASSWORD` and `LICENSE_FILE_PATH` values for the Curity Identity server*
-* run `make test`
-
-This, will run `prove` passing in the test or test directory (`t`). 
-
-Internet access to `hub.docker.com` is required for the `curity.t` test suite to pass, if the images required are not present locally.
-
-\* These variables can be set inline, in your environment or in the docker-compose.yaml file.
-
-NGINX must be in the system path; the tests will run the first `nginx` command that's found or bail if none is located. Also, the tests assume that the module is statically linked with NGINX. Before running them, be sure that the module is linked into the NGINX binary. Also, debug logging must be compiled into NGINX for some tests in `config.t` to pass. (This is the case if `nginx -V` includes `--with-debug` in the output.)
-
 ## Status
 This module is fit for production usage. 
 
+## Development Setup
+If you wish to build this module from source, in order to run against other NGINX versions, or to change the module's logic, see the [Development Wiki](https://github.com/curityio/nginx_phantom_token_module/wiki) for instructions.
+
 ## More Information
-For more information about Curity, its capabilities, and how to use it to issue phantom tokens, visit [curity.io](https://curity.io/product/token-service/#phantom_tokens). For background information on using Curity for API access, consult the [API integration section of the Curity developer manual](https://support.curity.io/docs/latest/developer-guide/api-integration/overview.html). For additional insights in how to apply this pattern to microservices and APIs, read _[How to Control User Identity within Microservices](http://nordicapis.com/how-to-control-user-identity-within-microservices/)_ on the Nordic APIs blog.
+For more information about Curity, its capabilities, and how to use it to issue phantom tokens for microservices, visit [curity.io](https://curity.io/product/token-service/?=use-cases?tab=microservices). For background information on using Curity for API access, see the [API security resources](https://curity.io/resources/api-security).
```

Comment:
this link with two question marks is probably not right

Reply:
Yeah, it looks super weird but it actually works and it's what we use for tabs.

Reply:
If it works, I wonder if it's something (the browser maybe) that is encoding that second `?` and  the`=` signs and if this will cause issues for others. 

I think you probably want `%3Duse-cases%3Ftab%3Dmicroservices`.

`https://curity.io/product/token-service/?%3Duse-cases%3Ftab%3Dmicroservices`